libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -79,10 +77,9 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
             var skillActivity = ObjectPath.Clone(dc.Context.Activity);
 
             skillActivity.DeliveryMode = dc.ActiveDialog.State[DeliverModeStateKey] as string;
-            var connectionName = dc.ActiveDialog.State[SsoConnectionNameKey] as string;
 
             // Just forward to the remote skill
-            var eocActivity = await SendToSkillAsync(dc.Context, skillActivity, connectionName, cancellationToken).ConfigureAwait(false);
+            var eocActivity = await SendToSkillAsync(dc.Context, skillActivity, DialogOptions.ConnectionName, cancellationToken).ConfigureAwait(false);
```

Comment:
Small cleanup: @gabog I pointed this out to @stevenic in my [review for JS](https://github.com/microsoft/botbuilder-js/pull/2177#pullrequestreview-405456308), but we don't need to pass around the ConnectionName as it's now bound to the instance of the Dialog through DialogOptions.

Reply:
Cleaned up.